.github/workflows/run-setup-script.yml

```diff
@@ -16,59 +16,64 @@ jobs:
   build-and-deploy:
     runs-on: ubuntu-latest
     steps:
-    # Checkout code
-    - uses: actions/checkout@main
-
-    # Log into Azure
-    - uses: azure/login@v1
-      with:
-        creds: ${{ secrets.AZURE_CREDENTIALS }}
-
     - name: Setup environment
       id: vars
       shell: bash
       run: |
-        echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
-        echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
-        MD5_OUTPUT="$(echo $(date +%s) | md5sum | awk '{print $1}')"
-        echo "::set-output name=ENV_CODE::$(echo pr${MD5_OUTPUT:0:7})"
+        echo "This is a dummy test"
+  #   # Checkout code
```

Comment:
Should we clean up this code before merging this PR? Remove commented code and text like "This is a dummy test",

Reply:
I am ok to remove, but I left it for now to leave the existing test code as is, so that if we want to revert back we still have the code handy to uncomment.

Reply:
As discussed, cleaned the file.